experiments/long_runs/snowy_land.jl

```diff
@@ -384,7 +384,7 @@ end
 function setup_and_solve_problem(; greet = false)
 
     t0 = 0.0
-    tf = 60 * 60.0 * 24 * 365
+    tf = 60 * 60.0 * 24 * 365 * 4
```

Comment:
I realized that we need to change the plotting! Right now it is assuming that we have one year, and are plotting indices [1,4,7,10] (indices refer to months). But we probably want to plot these in the last year.

We can remake the plots without rerunning the sim by looking here:
Using cached git repo /scratch/clima/slurm-buildkite/climaland-long-runs/2288/climaland-long-runs